packages/client/components/TaskFooterIntegrateMenu.tsx

```diff
@@ -33,7 +33,7 @@ const isIntegrated = (integrations: Integrations) => {
   const {atlassian, github, jiraServer} = integrations
   const hasAtlassian = atlassian?.isActive ?? false
   const hasGitHub = github?.isActive ?? false
-  const hasJiraServer = jiraServer.auth?.isActive ?? false
+  const hasJiraServer = jiraServer?.auth?.isActive ?? false
```

Comment:
JiraServer hasn't been added to the demo so we need to check if it exists here. I've created an issue to add GitLab to the demo. It'd be great to create one for Jira Server too if there isn't one already